### PR TITLE
Fix comment to associated MR when push pipeline

### PR DESCRIPTION
**Pull request by kitagry**
relate https://github.com/hirosassa/tfcmt-gitlab/pull/17

I discovered that the changes I made were blocked in this part( https://github.com/kitagry/tfcmt-gitlab/blob/main/pkg/notifier/gitlab/notify.go?plain=1#L67-L76 ).

### CODE REVIEW

File: pkg/notifier/gitlab/notify_test.go [hunk 1]
```diff
@@ -234,11 +234,10 @@ func TestNotifyNotify(t *testing.T) { //nolint:maintidx
 			exitCode: 2,
 		},
 		{
-			name: "apply case",
+			name: "get MR IID when MR number is 0",
 			createMockGitLabAPI: func(ctrl *gomock.Controller) *gitlabmock.MockAPI {
 				api := gitlabmock.NewMockAPI(ctrl)
-				api.EXPECT().ListCommits(gomock.Any()).Return([]*gitlab.Commit{{ID: "1"}, {ID: "2"}}, nil, nil)
-				api.EXPECT().ListMergeRequestsByCommit("2").Return([]*gitlab.MergeRequest{{IID: 1}}, nil, nil)
+				api.EXPECT().ListMergeRequestsByCommit("revision").Return([]*gitlab.MergeRequest{{IID: 1}}, nil, nil)
```
Comment (kitagry) [inline pkg/notifier/gitlab/notify_test.go:240]: I confirmed the behavior here.

**Comment by kitagry** [comment]
@hirosassa @yokomotod Please review!

File: pkg/notifier/gitlab/commits_test.go [hunk 2]
```diff
@@ -1,119 +1 @@
 package gitlab
```
Comment (yokomotod) [inline pkg/notifier/gitlab/commits_test.go:1]: we can remove this file ?

**Review by yokomotod — APPROVED** [review]
@kitagry You mean !17 have actually never worked, right ?

If I understood correctly, LGTM

**Pull request merged**